README.md

```diff
@@ -152,107 +152,108 @@ Documentation for the providers' settings can be found [here](docs/README.md "Pr
 
 If a provider you're looking for does not exist, consider making a PR to add one.
 
-| Provider | Stable | Nightly | Documentation |
```

Comment:
Please revert all these formatting changes and just add the new line at the appropriate point in the A-Z order. Otherwise it's just ongoing maintenance to adjust for alignment as new providers are added.